core/src/test/java/org/apache/iceberg/TestMetadataTableFilters.java

```diff
@@ -441,12 +441,6 @@ public void testPartitionSpecEvolutionRemovalV2() {
     assertThat(tasks).hasSize(expectedScanTaskCount(3));
   }
 
-  private DeleteFile posDelete(Table table, DataFile dataFile) {
```

Comment:
this addresses https://github.com/apache/iceberg/pull/11485#discussion_r1840262214